src/transformers/models/clip/image_processing_clip.py

```diff
@@ -124,8 +125,8 @@ def __init__(
         self.do_rescale = do_rescale
         self.rescale_factor = rescale_factor
         self.do_normalize = do_normalize
-        self.image_mean = image_mean if image_mean is not None else [0.48145466, 0.4578275, 0.40821073]
-        self.image_std = image_std if image_std is not None else [0.26862954, 0.26130258, 0.27577711]
+        self.image_mean = image_mean if image_mean is not None else IMAGENET_DEFAULT_MEAN
```

Comment:
I would be careful with this change. The default values here are not exactly the same values as those in `IMAGE_DEFAULT_MEAN`. Changing here will affect the pixel values of all models that use the CLIPImageProcessor. To make sure it's not going to affect things downstream, could you sanity check a CLIP integration test e.g. 

`RUN_SLOW=1 pytest tests/models/clip/test_modeling_clip.py::CLIPModelIntegrationTest::test_inference`